src/components/ConnectButton/index.tsx

```diff
@@ -1,8 +1,9 @@
+import { useWallet } from "@solana/wallet-adapter-react";
+import { WalletName } from "@solana/wallet-adapter-wallets";
```

Comment:
You can't rely on `WalletName` here. There's no guarantee that all the values in that enum are supported by the application. Only wallets passed into the context provider are valid selections.